src/styles/blocks/heder.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+
+  &__picture {
+    width: 40px;
+    height: 40px;
+    padding: 0;
+  }
+
+  &__navlist-linkstyle {
+    display: inline-block;
+    color: $main-text-color;
+    height: 60px;
+    line-height: 60px;
+    font-size: 12px;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    position: relative;
+    padding: 0;
+    transition: color 0.3s ease;
+  }
+
+  &__navlist {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__navlist-items {
```

Comment:
The class name `&__navlist-items` is not following the BEM naming convention. It should be written in lower case and separated by hyphens. So, it should be renamed to `&__nav-list-items`.